superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterDivider.tsx

```diff
@@ -124,7 +130,8 @@ const HorizontalOverflowDivider = ({
               display: block;
               font-size: ${theme.typography.sizes.s}px;
               color: ${theme.colors.grayscale.base};
-              margin: 0;
+              margin: ${theme.gridUnit * 2.5}px 0 0 0;
```

Comment:
I think that we should use full grid units. Can we change 2.5 to 2 or 3 (whichever looks better)

Reply:
Fixed in https://github.com/apache/superset/pull/22307!